src/Servers/HttpSys/src/NativeInterop/DisconnectListener.Log.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.HttpSys
+{
+    internal partial class DisconnectListener
+    {
+        private static class Log
+        {
+            private static readonly Action<ILogger, Exception?> _disconnectHandlerError =
+                LoggerMessage.Define(LogLevel.Error, LoggerEventIds.DisconnectHandlerError, "CreateDisconnectToken Callback");
+
+            private static readonly Action<ILogger, Exception?> _disconnectRegistrationCreateDisconnectTokenError =
+                LoggerMessage.Define(LogLevel.Error, LoggerEventIds.DisconnectRegistrationError, "CreateDisconnectToken");
+
+            private static readonly Action<ILogger, Exception?> _disconnectRegistrationError =
+                LoggerMessage.Define(LogLevel.Error, LoggerEventIds.DisconnectRegistrationError, "Unable to register for disconnect notifications.");
```

Comment:
There were two log events that shared an event Id but with different messages. I wasn't sure whether to unify them or not, so I left them distinct for now.

Reply:
Oh you found a problem. We should unify them.

Reply:
Or use different ids